integration_tests/features/steps/compare_images.py

```diff
@@ -11,7 +11,7 @@ def step_impl(context, source):
     new_source = ""
 
     for f in source.split(" "):
-        f = os.path.join(context.data_path, f)
+        f = os.path.join(os.environ['DATAPATH'], f)
```

Comment:
Why environment variable over context attribute?

Reply:
I need to pass it into the scripts being called, but I am going to use just a context attribute and then pass it in when the commands are called like so `'DATAPATH={} ...'.format(context.data_path)`